backend/data/degrees.js

```diff
@@ -0,0 +1,2671 @@
+module.exports = [
```

Comment:
Should we just include the prefiltered data?

Reply:
myeh...